modules/cache/tests/cache/RedisTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+include_once(Kohana::find_file('tests/cache', 'CacheBasicMethodsTest'));
```

Comment:
This include is sort of weird. I see that other cache test classes use it aswell, but i think with proper naming we should be able to use Kohana filesystem, right?

Reply:
Guess you are right...i just kept it there because as you said it is in other cache calsses used aswell

Reply:
Oh i see,
no this includeis neccessary, because `CacheBasicMethodsTest`doesn't inside `tests` folder doesn't follow kohana naming convetion